### ISSUE TITLE
Background Image missing when Data URI is specified

### ISSUE THREAD

**Issue by wei** [vercel/satori#304]
# Bug report

## Description / Observed Behavior

Background image does not work when data uri is applied.

## Expected Behavior

Background image should work when data uri is applied.

## Reproduction

[Works when https url is specified 
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/5880908/201492633-3ebf8f8a-f057-45fc-8618-51b30889028e.png">
 ](https://og-playground.vercel.app/?share=ZVLLTuswEP0Vy-gq90q9iaFpClbpgocEC1YgscnGjSeJwYkj2-mDqv_OuCGIx8rjc2bmzGtPCyOBcrqQap23hDi_03C53webkBpUVXtOolPG_kSTAdwo6esfmFSu02KHaKlhO6LBvlEWCq9Mi1xhdN-0Iyu0qtp7D40LFLQe7Ei99M6rcndtEGyD_nd6JYrXypq-lddGG4v8SVmWn6oY9ajegJPp2Rfo-aOXjLFfae4bUaF_1Fv9t_a-czxJrNjElfJ1v-od2GIoJS5Mk_iN8lgNvtCYF5U0woWvcA68S9y6Sk7LjE1jtP5FQepwyNtlMBYIfRniZU7ns5wOyFrB5spsEWOEkfmMZJ9UqbRG_IQxNkLjnkgjbKXCcENMt42OYoQc5VCwE74mEoMfpvN4dhGfzfQ0iy8we3oXZ-nw-Z-lbzldLpLgPRQa2hgsPIyfak-m4yRlKLW8A63NhDwbq-UiQV8MGl86oaYLq3eU7-mxY8rPcfx0uCvK0_CRsOoryr3tYUKPE33adeEmP-aLacL-bpsVyMHtcHgH)

[Does not work when base64 data URI of the same image is specified 
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/5880908/201492642-7dca1c88-d088-4d09-9cd9-76eb30bc23c5.png">
 ](https://og-playground.vercel.app/?share=lVTbbqNIEP0VxGiVXY036RhwJt4kEgbWJqKxHYgSIz8MBtzuNhfLgLlE-fethvVobg8zD6irT9U5VdVN15sYZGEkjsW7kJ7WqSDkRRNH929v3BaEXUTJrhgLF9cI_XEx6MGKhsXuOyyk-SH2G0C3cVSfUW7r9BgFBc1S8AVZXCbp2evHlKRmESU5d0VpER3PLlbmBd02WgZgyvN_6974wZ4cszINtSzOjuD_sN1uv2QFlkPbaCxIw6-gl_97GSH0g4yZ-ATiP5fH-M_QL_wx5cBVfiIf6yT-Z-Pn0UgeLGb20Gsm8ualLoMWUX_2hAI9O1lSKIWNIuFGOQVJcMJMrbB224ZJQM2Zd_BeQ20jkVuTqQRramtT_pkfF7PJLpwS4iVxvtERNZE3XSKMnjSTeLB3XTw0dUNetSqZO7Ji60Yx1-TKdvcEu6SAr1s7n6ZWlmvIlktKrD8rvW3IS87lHKrKFlNBD2rgvClWLLZH2FErnsPUOW_V-1xCrVb-5L3Uuf86aT1nkvhgLxyT2WyFbBc4dMIiqBG7Rmk7JllJe-jPaCz2DD0_yhB7thWwr7n9y5ps-VuawfRf5Gs_1dsvnEe764k9MsvFpc3MFs6gsKlcYRYUFls2mCJ-XmCvpLmDSuzi2nJNuCvUmDpfVWTqUJO7vzZ1fvYG3Ic6BG7Fzx3OD-KfOQ53we_BLLAjt5gZBaZyjVvYU9QAXs7doIW6pbmGoH4s2Y0KXNB8zX7ay9Pem577AO0GS6Cmd_lrqAXyqF3tmKqI69ga72UJdSAUUfO0YNUpkLx0Qe7v__rM__v393X6wI07-L2_etH3a_FGWYs9cqJRNclqwJCAhBtFGH1xbWkcA_4BIXSGzkNDSPwjofylc86hvuiSCUKXDhIe_GInhEDG0s2lcns5VGJpdHkL6vLsciT3m79HcrsWH-6ueHRfKH-IvQVT6vtsbnYYCzKCVA-zKI6zgfCSHePw7gpigXRexYGYHfgcysXxm9h1LI4_wSwQ-yEnjmW-CaNNScRxcSyjgRglGaNuc-ADsqi6HcjwYWIkmyjsw97f_wM)

## Additional Context

Satori version, and any other context about the problem here.

`"satori": "^0.0.44"`

**Comment by sahithyandev** [comment]
Is this not working for only SVGs? or for all image types?

**Comment by wei** [comment]
> Is this not working for only SVGs? or for all image types?

@sahithyandev It doesn't work for pngs either. So I believe this is for all image types using data URI.

[png https url 
<img width="1770" alt="image" src="https://user-images.githubusercontent.com/5880908/201836476-cfe83b36-60f4-496b-b6bb-0586727e6ff7.png">
 ](https://og-playground.vercel.app/?share=ZVJNb9swDP0rgorBG5DZauLYq5DmsG5Ae9hpBXrxRbFkWa0sGZKcjwb576XiuujHSeR7JB9F8ohrywWmeMXVtjII-XDQ4vp4jDZCrVCyDRQll4R8S2YjuFM8tJ8wrnyv2QHQRov9hEb7j3KiDsoa4Gqrh85MLNNKmrsgOh8pYYJwE_U4-KCaw40F0ET9j_SG1U_S2cHwG6utA_6iaZo3Vcj6r54FRYv5O-jh9S8FIV_K3HVMQnwyOP29DaH3NMsc26VShXbYDF64emwlrW2XhZ0K0A28orOPKuuYjy7zXgSflfN9Oc8um4Is0t7IH0kUO50qs47Gym_luzFeV7hcVnhEtkrsfts9YAQRVC5R8UY1SmvALwghEzRtCnXMSRXHG3P6fXIWQ-gsB4I9Cy3ikPxvUabLq3S-1IsivYLq-W1a5KPzs8ifK7xeZTF6bDSDTkcLTuOz2r3tKcoJSK1vhdZ2hh6s03yVQSwkTS-eYdvH5XtMj_j8Y0x_wQLweFmY5tHhYjNITIMbxAyfZ3p_6ONVvk4YysQN_u02go9hp9ML)

[png data uri 
<img width="1771" alt="image" src="https://user-images.githubusercontent.com/5880908/201836656-ec535a9a-329a-480e-98f4-28fb5ca66034.png">
 ](https://og-playground.vercel.app/?share=lVXZjqNGFP0Vy6OoE9EZFgM2Ts9IrDZ2g9kxVr9gs7gwZt9b_e-B7vRoMnlKSVCnzrm3TpUQ977OL6nnz9fzJw80L8lsVlZ97H97fZ3wbHb1QXit1rMHFEF-e3j8IFvgVddfOA-UWez2IxvEfvfJTpgDhX-pQJqM2iWN63vyqboxCBOx8u_lJPlJ5RefUlSXFQh6Nh3JZPL_t3x2L7ewSOvEY9M4LUb9SxAEP1zHLB0M_nq2wH6i7H_uQiLIf7YR7244xj_URfy751buGkwEnCXhX2e39En8EVjMQWuR_SZM6XHIunnlzXBE_PRiRJaWphlIiIxOAVeUkSze2l74AR74xf98sCxkd3dN5coaOQQ-s0A7PtrcDZY7hktQDvApjzLCCnP73IEB6IGSRrZ06g6rm96UpAyfkDM0pClrByjLuRxDHFkOXUHwNMBCglZFCkkezwYAY1pT4zb1CVY6l2oN1ewax9xNd-ASTTdiiRY3rNJQMGzu4EbdQ_SANpd-044RO0M3Ncbahgyx04S4VFuelyPzOmwglEJtCcX2S_bcEHZwFLqbJJ-JkINYj3LMQ3-NWka6xlV1ifa1mm535HFrc888Vmma0sXgyqOWo-ZytsCPqkqaPWZp1i2P9oYQuZrBUkCENy2H38kIIm8o4hcmfvcUxThYWy9X2pbSBTJb1n7mLwZqudP0OtGZrt5seV1vKinhsxGzI7bYswYdE9xp3XC_4yHZJxxo2zKyeFzkx_sh6eJUb-Sj6PXDqrntreBe3UqJals7BU3Z5Sl-tqz-oCKnS7yNTMBI6hDR-6UcayIRsXQZnDk0xAKYoi8hvQppqVCjrmiOVGbbmJrYPo8cIto5lhhygyqHVb1gJYh4tToxp471hVwb0LNElwK6EcF1S3gLwVA3_cKAScLSKqeCGJS-IpdOrI-AMIAIqhCc5PJG63tLJuSDGAmOHvILsdrCIr0oe4iUnkvd0fdtQTmDQivqFkI2HLcNGBYqYay5mm51hn17D0oyd4UYIQyrN5-bgCfbbI9haKMX7QlVrMHgosCkXC6niBNidyeSUM2stA2zXomi3qtVL4-fxFfa5Ngh2PKQUE1QAo5xJHK5OlSm47atxx8NW990qV54EmazXPYs3zvHK0ybxsnTksqlm1rcjj1uFJR0bR20UiAjV-EiuOQV-xwkkpdGWSMuTgEmwBDukjokKNtKysngTlMYnPCLfWlDrCRskDMQT4mCO4R7ai-NVCsHDoI7ReD6lJBwca-3F1sR6FxaRJsASw-Hi0HJLv5c6JbNHiohCraUSgtqw-xFdNjxOYexFsUzbrShMw2l_YJjVIVO2S7hi8yZ_iWajwXjptfqnWX_eJjqz9vbS_J9Ak9lE_5UWb-9zJfEy_yDaYDfMmk3csgMmS2JGflDCkAcj_wXBEE-qc_iPbu7RQimijvlZN3Du9ls9m43GmZudZ15Y7K0WH4lqK8YES_Ir9S4O779SuIfiz9JfHiZf3-Cp-iPg8LjST_Q2C1-dTPSbD3DkdHq-9aP4_RxZqdF7D3BY-yY9DnPH-dpNvWDcr5-nb_feL5ejTV5_tFs5mt8Wnj-uQ7n66qo_ce5f08jYPTZ1Kiq9n01bjMVdf5-9r2PsLe3vwE)

**Comment by sahithyandev** [comment]
I hope I have fixed the issue. Test it out [here](https://satori-playground-git-fork-sahithyandev-fix-background-i-7197fe.vercel.sh/?share=lVRbb6M4FP4riNGqu5ps6wZIp2xbiQCbUGGSFKo2KA9DwHFMuEQBwqXqf99jshnN7WHmAfn4O_6-czE-b2KYR0RUxbuIHVeZIBRlm5D7tzduC8KWMLotVeHiGqE_LgYnsGZRuf0Oi1ixT4IW0E1CmjPKbYMdSFiyPANfmCdVmp29QcJoZpUkLbiLZCU5nF1xVZRs0-o5gBmP_617HYQ7esirLNLzJD-A_8Nms_kSFVgu64gqSMOvoJf_axkh9IOMlQYUzn-uDsmfUVAGKuPAVXGkH5s0-WcdFGQkD-ZTZ-i3Y3n90lRhh1gwfUKhkR9tKZKiVpFwqxzDNDziWKuxfttFacisqb_3XyN9LdFbK9Yo1rXOYfyzPs6n4200odRPk2JtIGYhf7JAGD3pFvVh73l4aBmmvOw0OnNlxTHMcqbLtePtKPZoCV-_9j5dq23PlG2PVth4Vk62KS84l3OYJtuxBnqQA-dNsGLHO4RdreYxLIPzliefR5ndyZ_8l6YIXsed747TAOy5a8VOvESOBxw2jgnkiD2zclyLLqUd1Ge2dvwMNT_KcPZsK2Bfc_uXNePFb2mGk39RoP9Ubzd3H52-pvgxtj1cObHVQQ9Kh8k1jsPSjhctZoj3C-ylNHNRhT3c2J4Fd4Vay-CrhiwDcvJ215bBe2_CfWhD4Na879A_OP_McbgLfg9WiV25w7FZYiY3uIM9Qy3g1cwLO8hbmukI8seS02rABc3X_Ke1PO38ybkO0G6xBGpGH7-BXCCO1ueOmYa4jqPzWhaQB0KEWcd5XB9Dyc_m9P7-r8_8v39_X2UP3LiD3_urF32_Em-UlXhCjozU47wBDAlIuFGE0RfXhiUJ4B8QQmfoPDSENDhQxl865-ybiz6YIPThIOA-KLdCBGQs3Vwqt5dDJZFGl7egLk8vR_Jp8_dI7lbiw90VP31KlD_EkwVT6vtoXr5XBRlBqIcpSZJ8ILzkhyS6u4KzQDqv4kDM93wOFaL6JvYVi-onmAXiaciJqsw3EVlXVFQ3QVKQgUjSPGZeu-cTsqz7HejwaWKmaxKJanmoyPv7fw)

**Comment by wei** [comment]
@vchirikov the data uri you provided is not a valid jpeg. It's a png mistyped as jpeg.

```
data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAQAAAAEACAIAAADTED8xAAADMElEQVR4nOzVwQnAIBQFQYXff81RUkQCOyDj1YOPnbXWPmeTRef+/3O/OyBjzh3CD95BfqICMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMK0CMO0TAAD//2Anhf4QtqobAAAAAElFTkSuQmCC
```

It works fine once you convert the file to a jpg first before generating a data uri, e.g.:

```
data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD//gAfQ29tcHJlc3NlZCBieSBqcGVnLXJlY29tcHJlc3P/2wCEAAQEBAQEBAQEBAQGBgUGBggHBwcHCAwJCQkJCQwTDA4MDA4MExEUEA8QFBEeFxUVFx4iHRsdIiolJSo0MjRERFwBBAQEBAQEBAQEBAYGBQYGCAcHBwcIDAkJCQkJDBMMDgwMDgwTERQQDxAUER4XFRUXHiIdGx0iKiUlKjQyNEREXP/CABEIAQABAAMBIgACEQEDEQH/xAAWAAEBAQAAAAAAAAAAAAAAAAAECQj/2gAIAQEAAAAAwMlSVJUlSVJUlKkqSpKkqSpKkpUqb6kqSpKkqSpKkpUlSVJUlSUqUlKpwJUlSVJUlSVJUlKkqSpKkqSlSkpnApKkqSpKkqSpKkpUlSVJUlSUqUmcCVJUlSVJUlSVJUlKkqSpKkqSlSpvqSpKkqSpKkqSpKkpUlSVJUlSUqnAlSVJUlSVJUlSVJUlKkqSpKkqSmcCkqSpKkqSpKkqSpKkpUlSVJUlSZwJUlSVJUlSVJUlSVJUlKkqSpKkqm+pKkqSpKkqSpKkqSpKkpUlSVJUmcCVJUlSVJUlSVJUlSUqUlKkqSpKpvqSpKkqSpKkqSpKkqSlSkpUlSVJnAlSVJUlSVJUlSVJUlSUqUlKkqSqb6kqSpKkqSpKkqSpKVKSlSkpUlSZwJUlSVJUlSVJUlSVJSpSUqUlKkqnAlKkqSpKkqSpKkpUpKVKSlSkpUmcCkpUlSVJUlSVJUlKlJSpSUqUlKpwJUlKkqSpKkqSpKkpUpKVKSlSkpnApKkpUlSVJUlSVJUlKlJSpSUqUmcCVJUlKkqSpKkqSpKkpUpKVKSlSpvqSpKkpUlSVJUlSVJUlKlJSpSUqnAlSVJUlKkqSpKkqSpKkpUpKVKSmcCkqSpKkpUlSVJUlSVJUlKlJSpSZwJUlSVJUlKkqSpKkqSpKkpUpKVKm+pKkqSpKkpUlSVJUlSVJUlKlJSqcCVJUlSVJUlKkqSpKkqSpKkpUpKZwKSpKkqSlSkpUlSVJUlSVJUlKlJnAlSVJUlSUqUlKkqSpKkqSpKkpUqb6kqSpKkqSlSkpUlSVJUlSVJUlKpwJUlSVJSpSUqUlKkqSpKkqSpKkpnApKkqSpKVKSlSkpUlSVJUlSVJUmcCVJUlSVJSpSUqUlKkqSpKkqSpKv/EABYBAQEBAAAAAAAAAAAAAAAAAAQIAP/aAAgBAhAAAADEKQhSEKQhCkIWqyEKQhSEKQhCkJVhCEKQhSEKQhCkqspCEKQhSEKQhC1WQpCEKQhSEKQhKsIQpCEKQhSEKQlVlIQpCEKQhSEKSqyFIQpCEKQhSELVZCFIQpCEKQhSEqspCFIQpCEKQhSVWQpCFIQpCEKQharIQpCFIQpCEKQlWEIQpCFIQpCEKSqykIQpCFIQpCELVZCkIQpCFIQpCEqwhCkIQpCFIQpCf//EABYBAQEBAAAAAAAAAAAAAAAAAAQIAP/aAAgBAxAAAADJUlKkpUlKVJSqakpUlKkpUlKVJTNSkpUlKkpUlKVJmpKkpUlKkpUlKVTUlKkpUlKkpUlKZrSlKkpUlKkpUlM1KSlKkpUlKkpUmakqSlKkpUlKkpVNSUqSlKkpUlKkpmpSUqSlKkpUlKkzUlSUqSlKkpUlKpqSlSUqSlKkpUlM1KSlSUqSlKkpUmakqSlSUqSlKkpVNSUqSlSUqSlKkpmpSUqSlSUqSlKk/wD/xAAUEAEAAAAAAAAAAAAAAAAAAAAC/9oACAEBAAECACQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQSQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQQf/EABQQAQAAAAAAAAAAAAAAAAAAAKD/2gAIAQEAAz8AAB//xAAUEQEAAAAAAAAAAAAAAAAAAAAC/9oACAECAQECAGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGGP/EABQRAQAAAAAAAAAAAAAAAAAAAID/2gAIAQIBAz8AAH//xAAUEQEAAAAAAAAAAAAAAAAAAAAC/9oACAEDAQECACwwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwywwwwwwwwwwwwwwwwy//EABQRAQAAAAAAAAAAAAAAAAAAAID/2gAIAQMBAz8AAH//2Q==
```

**Comment by wei** [comment]
> I hope I have fixed the issue. Test it out [here](https://satori-playground-git-fork-sahithyandev-fix-background-i-7197fe.vercel.sh/?share=lVRbb6M4FP4riNGqu5ps6wZIp2xbiQCbUGGSFKo2KA9DwHFMuEQBwqXqf99jshnN7WHmAfn4O_6-czE-b2KYR0RUxbuIHVeZIBRlm5D7tzduC8KWMLotVeHiGqE_LgYnsGZRuf0Oi1ixT4IW0E1CmjPKbYMdSFiyPANfmCdVmp29QcJoZpUkLbiLZCU5nF1xVZRs0-o5gBmP_617HYQ7esirLNLzJD-A_8Nms_kSFVgu64gqSMOvoJf_axkh9IOMlQYUzn-uDsmfUVAGKuPAVXGkH5s0-WcdFGQkD-ZTZ-i3Y3n90lRhh1gwfUKhkR9tKZKiVpFwqxzDNDziWKuxfttFacisqb_3XyN9LdFbK9Yo1rXOYfyzPs6n4200odRPk2JtIGYhf7JAGD3pFvVh73l4aBmmvOw0OnNlxTHMcqbLtePtKPZoCV-_9j5dq23PlG2PVth4Vk62KS84l3OYJtuxBnqQA-dNsGLHO4RdreYxLIPzliefR5ndyZ_8l6YIXsed747TAOy5a8VOvESOBxw2jgnkiD2zclyLLqUd1Ge2dvwMNT_KcPZsK2Bfc_uXNePFb2mGk39RoP9Ubzd3H52-pvgxtj1cObHVQQ9Kh8k1jsPSjhctZoj3C-ylNHNRhT3c2J4Fd4Vay-CrhiwDcvJ215bBe2_CfWhD4Na879A_OP_McbgLfg9WiV25w7FZYiY3uIM9Qy3g1cwLO8hbmukI8seS02rABc3X_Ke1PO38ybkO0G6xBGpGH7-BXCCO1ueOmYa4jqPzWhaQB0KEWcd5XB9Dyc_m9P7-r8_8v39_X2UP3LiD3_urF32_Em-UlXhCjozU47wBDAlIuFGE0RfXhiUJ4B8QQmfoPDSENDhQxl865-ybiz6YIPThIOA-KLdCBGQs3Vwqt5dDJZFGl7egLk8vR_Jp8_dI7lbiw90VP31KlD_EkwVT6vtoXr5XBRlBqIcpSZJ8ILzkhyS6u4KzQDqv4kDM93wOFaL6JvYVi-onmAXiaciJqsw3EVlXVFQ3QVKQgUjSPGZeu-cTsqz7HejwaWKmaxKJanmoyPv7fw)

It works great!!! I left a comment about supporting utf8 data uris in https://github.com/vercel/satori/pull/310#discussion_r1022627668

**Comment by wei** [comment]
> I hope I have fixed the issue. Test it out [here](https://satori-playground-git-fork-sahithyandev-fix-background-i-7197fe.vercel.sh/?share=lVRbb6M4FP4riNGqu5ps6wZIp2xbiQCbUGGSFKo2KA9DwHFMuEQBwqXqf99jshnN7WHmAfn4O_6-czE-b2KYR0RUxbuIHVeZIBRlm5D7tzduC8KWMLotVeHiGqE_LgYnsGZRuf0Oi1ixT4IW0E1CmjPKbYMdSFiyPANfmCdVmp29QcJoZpUkLbiLZCU5nF1xVZRs0-o5gBmP_617HYQ7esirLNLzJD-A_8Nms_kSFVgu64gqSMOvoJf_axkh9IOMlQYUzn-uDsmfUVAGKuPAVXGkH5s0-WcdFGQkD-ZTZ-i3Y3n90lRhh1gwfUKhkR9tKZKiVpFwqxzDNDziWKuxfttFacisqb_3XyN9LdFbK9Yo1rXOYfyzPs6n4200odRPk2JtIGYhf7JAGD3pFvVh73l4aBmmvOw0OnNlxTHMcqbLtePtKPZoCV-_9j5dq23PlG2PVth4Vk62KS84l3OYJtuxBnqQA-dNsGLHO4RdreYxLIPzliefR5ndyZ_8l6YIXsed747TAOy5a8VOvESOBxw2jgnkiD2zclyLLqUd1Ge2dvwMNT_KcPZsK2Bfc_uXNePFb2mGk39RoP9Ubzd3H52-pvgxtj1cObHVQQ9Kh8k1jsPSjhctZoj3C-ylNHNRhT3c2J4Fd4Vay-CrhiwDcvJ215bBe2_CfWhD4Na879A_OP_McbgLfg9WiV25w7FZYiY3uIM9Qy3g1cwLO8hbmukI8seS02rABc3X_Ke1PO38ybkO0G6xBGpGH7-BXCCO1ueOmYa4jqPzWhaQB0KEWcd5XB9Dyc_m9P7-r8_8v39_X2UP3LiD3_urF32_Em-UlXhCjozU47wBDAlIuFGE0RfXhiUJ4B8QQmfoPDSENDhQxl865-ybiz6YIPThIOA-KLdCBGQs3Vwqt5dDJZFGl7egLk8vR_Jp8_dI7lbiw90VP31KlD_EkwVT6vtoXr5XBRlBqIcpSZJ8ILzkhyS6u4KzQDqv4kDM93wOFaL6JvYVi-onmAXiaciJqsw3EVlXVFQ3QVKQgUjSPGZeu-cTsqz7HejwaWKmaxKJanmoyPv7fw)

@sahithyandev It works great!!! I left a comment about supporting utf8 data uris in https://github.com/vercel/satori/pull/310#discussion_r1022627668

**Comment by wei** [comment]
I noticed the `backgroundImage` data uri works fine when `backgroundSize` is specified. I know `<img>` tags require size to be specified in the attribute so adding `backgroundSize` will actually resolve my use-case.

[<img width="1772" alt="image" src="https://user-images.githubusercontent.com/5880908/201919212-433abbe5-0b1d-499c-a775-262ff9d7a441.png">
](https://og-playground.vercel.app/?share=lVTbbqNIEP0VxGiVXY036RhwJt4kEgbWJqKxHYgSIz8MBtzuNhfLgLlE-fethvVobg8zD7irT1WdU9XtrjcxyMJIHIt3IT2tU0HIiyaO7t_euC0Iu4iSXTEWLq4R-uNi0IMVDYvdd1hI80PsN4Bu46g-o9zW6TEKCpql4AuyuEzSs9ePKUnNIkpy7orSIjqeXazMC7pttAzAlOt_6974wZ4cszINtSzOjuD_sN1uv6hClkPbaCxIw6-gl_97GSH0A42Z-ATiP5fH-M_QL_wx5cBVfiIf6yT-Z-Pn0UgeLGb20Gsm8ualLoMWUX_2hAI9O1lSKIWNIuFGOQVJcMJMrbB224ZJQM2Zd_BeQ20jkVuTqQRramtT_pkfF7PJLpwS4iVxvtERNZE3XSKMnjSTeLB3XTw0dUNetSqZO7Ji60Yx1-TKdvcEu6SAr1s7n6ZWlmvIlktKrD8rvW3IS57Lc6gqW0wFPqiB502xYrE9wo5acQ1T53mr3ucSarXyJ--lzv3XSes5k8QHe-GYzGYrZLuQQycsghqxa5S2Y5KVtIf-jMZiz9DzowyxZ1sB-5rbv8zJlr_FGUz_Rb72U779wnm0u57YI7NcXNrMbOEMCpvKFWZBYbFlgyni5wX2Spo7qMQuri3XhLtCjanzVUWmDjW5-2tT52dvwH2oQ8it-LnD-UH8M8fhLvg9mAV25BYzo8BUrnELe4oawMu5G7RQtzTXENSPJbtRIRc4X7Of9vK096bnPoC7wRKw6Z1-DbWAjtrVjqmKOI-t8V6WUAdCETVPC1adAslLF-T-_q_PP_zx-3fCn_KhFrrfCx7y_r5OH7hxB0_gq1d_vxZvlLXYIycaVZOsBgwJSLhRhNEX15bGMeAfEEJn6DxYhMQ_EsqnAc8BvU5MEDo5EDz4xU4IIRlLN5fK7eVQiaXR5S2wy7PLkdxv_h7J7Vp8uLvi0X2h_LH2Fkyy79Xc7DAWZARSD7MojrOB8JId4_DuCmIh6byKAzE78FmVi-M3setYHH-CeSH2g1Acy3wTRpuSiOPiWEYDMUoyRt3mwIdoUXU7oOEDx0g2UdiHvb__Bw)

**Comment by wei** [comment]
Fixed in #310
